records_mover/records/targets/table/target.py

```diff
@@ -116,6 +116,14 @@ def can_move_from_temp_loc_after_filling_it(self) -> bool:
             return False
         return loader.has_temporary_loadable_directory_loc()
 
+    def temporary_loadable_directory_scheme(self) -> str:
+        driver = self.db_driver(self.db_engine)
+        loader = driver.loader()
+        if loader is None:
+            raise TypeError("Please check can_move_from_temp_loc_after_filling_it() "
+                            "before calling this")
```

Comment:
This isn't the ideal from a type safety perspective, but I looked at a few alternatives and couldn't come up with a better approach without making the move() method much uglier.

Reply:
With the understanding that this not shown to the External API, and mostly as a signal to fix logic elsewhere, I see no reason to belabor this point.